src/server.ts

```diff
@@ -72,6 +72,17 @@ const build = (opts: FastifyServerOptions): FastifyInstance => {
   checkRedisConnection();
 
   fastify.addHook('preValidation', async (req: FastifyRequest, reply: FastifyReply) => {
+    if (req.routeOptions.url?.startsWith('/api') && req.method === 'POST') {
```

Comment:
Shouldn't this be for all request methods?

Reply:
this is checking for credentials not all requests have credentials
only api call come with credentials

Reply:
Shouldn't the check you implemented apply for all api calls? Why limit it to `POST` calls only?

Reply:
so you can call this endpoint in a browser to view config

Reply:
if this is specific to a single endpoing, why isn't it in the `route` file. why does it have to awkwardly be in this hook?